app/models/models.py

```diff
@@ -0,0 +1,75 @@
+"""classes for test objects"""
+from flask import jsonify
+
+class User(object):
+	"""user class"""
+	def __init__(self):
```

Comment:
When you create a user, the object should have the name and the other user properties.